packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -49,6 +49,13 @@ export function getZoweDir(): string {
     return zowe.getZoweDir();
 }
 
+export function getFullPath(anyPath: string): string {
+    if (os.platform() === "win32") {
+        return fs.realpathSync.native(anyPath);
+    }
+    return fs.realpathSync(anyPath);
+}
+
```

Comment:
My suggestion on this function is to add a try/catch block to catch any possible errors this might through @phaumer

Reply:
So far we never saw this one crashing. This problem was so far only that the method itself was not found.